### PR TITLE
popover works, styling is still not showing under elements but is und…

### LINKED ISSUE #4874: CSS breaking in multiple subscription button
Content is not being displayed in the popover and the button styling of the Subscribe+ button has disappeared as well, however, the code is fine I believe because it's the same code that was working perfectly some time ago. I feel that this could be happening because of the changes in the Bootstrap version.
image

Please look into it if it's because of the Bootstrap version. Thanks a lot!

### PR DESCRIPTION
**Pull request by Chrisyee22**
…er.btn-default

Fixes #4874 (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x ] PR is descriptively titled 📑
* [x ] screenshots/GIFs are attached 📎 in case of UI updation
* [ x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

screen shot 2019-03-03 at 5 29 52 pm

### CODE REVIEW

**Comment by Chrisyee22** [comment]
@publiclab/reviewers would you be able to have a look at why the element.syle is showing blank when inspecting? I reset developer tools and checked the css link and everything looks good. The button styling info is below and I assume that is just due to how bootstrap works. Please let me know if I missed anything. 😄

File: app/views/home/subscriptions.html.erb [hunk 1]
```diff
@@ -46,7 +46,7 @@
...
+        <a class="btn btn-default btn-xs" rel="popover"  data-toggle="popover" data-placement="left" title="<%= t('home.subscriptions.share_subscription_link') %>" data-html="true" data-content="<input id='share-<%= subscription.tid %>' onClick='$(input_<%= subscription.tid %>).select()' type='text' value='https://<%= ActionController::Base.default_url_options[:host] %>/subscribe/tag/<%= subscription.tagname %>'/><div style='margin-top:10px;'><a style='padding-left:3px;' href='https://twitter.com/share' class='twitter-share-button'>Tweet</a></div> <script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0];if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src='//platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document,'script','twitter-wjs');</script>
```
Comment (jywarren) [inline app/views/home/subscriptions.html.erb]: Just checking here -- is the only change adding `data-toggle="popover"`? Thanks!

**Comment by jywarren** [comment]
This looks good to me; thanks! Asked a question inline; also, I think you've accidentally deleted `config/database.yml.sqlite.example`; could you add that back in? 

With those changes, this should be good to go; thank you!

PS - i'm not sure about the inspection issue; the odd thing about popovers is that they're generated by JavaScript so they're not originally in the DOM. But they should be existing properly by the time the inspector looks at them. So that's a bit mysterious. But if it works, that's fine with me! Thanks!

**Comment by jywarren** [comment]
Great work here!!!! :tada: :smile:

**Pull request merged**